src/test/java/org/springframework/social/salesforce/api/impl/ApiTest.java

```diff
@@ -157,6 +166,23 @@ public static void testChatterOperations(Salesforce template) {
         System.out.println(template.chatterOperations().updateStatus("Updated status via #spring-social-salesforce!"));
     }
 
+    public static void testCommunityOperations(Salesforce api) {
```

Comment:
This is an Ok test...we need tests added to add tests like we see here https://github.com/jottley/spring-social-salesforce/blob/master/src/test/java/org/springframework/social/salesforce/api/impl/SObjectsTemplateTest.java for the connectTemplate

Reply:
Hi Jared, I added a test for this here: https://github.com/flv22/spring-social-salesforce/commit/f20c9ea9e7461be2d5f270ad8bc73fce5d0d74b2